Packs/Elasticsearch/Integrations/Elasticsearch_v2/Elasticsearch_v2.yml

```diff
@@ -21,6 +21,14 @@ configuration:
   name: proxy
   required: false
   type: 8
+- defaultvalue: Elasticsearch
```

Comment:
Please add additional info here explaining who will need this and when

Reply:
```suggestion
- defaultvalue: Elasticsearch
  additionalInfo: In some hosted ElasticSearch environments (AWS), the standard ElasticSearch client is not supported. If your environment is not supported, please select OpenSearch as the parameter.
```